### PR TITLE
Minor changes to metrics pagination and removal of orphaned rule

### PR DESCRIPTION
**Pull request by sg00dwin**
Screen Shot 2020-07-22 at 4 40 09 PM
Screen Shot 2020-07-22 at 4 07 16 PM

### CODE REVIEW

File: frontend/public/components/monitoring/_monitoring.scss [hunk 1]
```diff
@@ -324,13 +324,6 @@ $monitoring-line-height: 18px;
   }
 }
 
-.query-browser__pagination {
```
Comment (spadgett) [inline frontend/public/components/monitoring/_monitoring.scss:327]: Looks like this is still used here: https://github.com/openshift/console/blob/master/frontend/public/components/monitoring/metrics.tsx#L598

Comment (sg00dwin) [reply frontend/public/components/monitoring/_monitoring.scss:327]: Removed since it looks to be no longer needed.

**Comment by spadgett** [comment]
/cc @kyoto 

It would be good to get @kyoto to review since he would know why that class was added originally.

**Comment by spadgett** [comment]
Although I see @rhamilto tagged this as I was writing that comment, and I trust his review :)

**Pull request merged**